lib/components/molecules/Popover/tests/Popover.test.js

```diff
@@ -0,0 +1,15 @@
+import { shallow } from 'enzyme';
+// import renderer from 'react-test-renderer';
+
+import Popover from '../index';
+
+describe('<Popover />', () => {
+  let PopoverComponent = '';
+  beforeEach(() => {
+    PopoverComponent = shallow(<Popover>Test</Popover>);
+  });
+
+  test('should render correctly', () => {
```

Comment:
Test cases needs to improve, you have a open/close functionality too. Test it here!